qir/qat/Version/version_builder.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import sys
+from string import Template
+
+version_file, template, output = sys.argv[1:]
+with open(version_file, "r") as fb:
+    values = [x.strip().split(" ", 1) for x in fb.readlines() if " " in x.strip()]
+    kwargs = dict(values)
```

Comment:
I assume that the `kwargs` is a _global_ var here, rather than _local_.

Reply:
Yes, thatt is correct.